src/jdk.jcmd/share/man/jcmd.1

```diff
@@ -178,58 +195,71 @@ Prints all compiled methods in code cache that are alive.
 Impact: Medium
 .RE
 .TP
-\f[V]Compiler.perfmap\f[R] [\f[I]arguments\f[R]] (Linux only)
-Write map file for Linux perf tool.
+\f[V]Compiler.directives_add\f[R] \f[I]filename\f[R] \f[I]arguments\f[R]
+Adds compiler directives from a file.
```

Comment:
I might be misreading this one, it looks like there are two arguments but the the help output shows just one, maybe check that one.

Reply:
I didn't spot anything else, the update looks good to me.